core/src/main/java/org/mule/DefaultMuleEvent.java

```diff
@@ -118,15 +121,26 @@ public DefaultMuleEvent(MuleMessage message,
             .getDefaultResponseTimeout(), null, null);
     }
 
+    public DefaultMuleEvent(MuleMessage message,
```

Comment:
We have javadoc on some constructors and others now. I'm not sure how valuable javadoc which says "creates MuleEvent" on each constructor is though TBH.  I tend to use javadoc on interfaces always, but in implementations only when method signature is not already self-explanatory.